src/Helper/FileStreamer.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace eLife\Journal\Helper;
+
+use GuzzleHttp\Client;
+use Psr\Http\Message\ResponseInterface;
+use RuntimeException;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpFoundation\StreamedResponse;
+use Symfony\Component\HttpKernel\Exception\HttpException;
+
+final class FileStreamer
+{
+    private $client;
+
+    public function __construct(Client $client)
+    {
+        $this->client = $client;
+    }
+
+    public function getResponse(Request $request, string $uri) : Response
+    {
+        $xForwardedFor = array_filter(array_map('trim', explode(',', $request->isFromTrustedProxy() ? $request->headers->get('X-Forwarded-For') : '')));
+        $xForwardedFor[] = $request->server->get('REMOTE_ADDR');
+
+        /** @var ResponseInterface $fileResponse */
+        $fileResponse = $this->client->request('GET', $uri, [
```

Comment:
Should change this name now we're not necessarily forcing a download. `$backendResponse` maybe?